specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -460,6 +460,64 @@
         }
       }
     },
+    "/keys/{keyName}/restore": {
+      "put": {
```

Comment:
estore single key from single key backup uses a "POST" not a "PUT", why are you proposing to use a PUT for selective restore?
https://docs.microsoft.com/en-us/rest/api/keyvault/restorekey/restorekey

Reply:
As we have discussed over the teams conversation, I kept the PUT method to indicate that even if the key to be restored was deleted then also we would create the key.